submissions/Chikus/port-sniffer/sniffer.js

```diff
@@ -0,0 +1,69 @@
+const Net = require('net');
+var host = '';
+var lower_port = 0;
+var higher_port = 65535;
+var port_list = [];
+var timeout = 300;
```

Comment:
No need to use var if you are not looking for exactly function-wide scope. It's better to be consistent and use let instead. Or maybe try to reduce amount of mutable global variables by passing them as arguments from function-to-function.

Reply:
Done, and thanks

Reply:
Sure, Done, thanks!